torch/_inductor/kernel/flex_attention.py

```diff
@@ -65,19 +65,14 @@ def index_to_other_buffers(cnt: int, graph_type: SubgraphType) -> int:
     #   key,
     #   value,
     #   score_mod,
-    #   sparse_kv_num_blocks,
-    #   sparse_kv_indices,
-    #   sparse_q_num_blocks,
-    #   sparse_q_indices,
-    #   SPARSE_KV_BLOCK_SIZE,
-    #   SPARSE_Q_BLOCK_SIZE,
+    #   block_mask,
     #   *other_buffers
     # ]
     # For fwd_graphs we have 5 dummy values this when the first lifted args
-    # is seen cnt = 5 and the start of the index_buffers is at args[10]
-    # thus we add 5 from the current cnt
+    # is seen cnt = 5 and the start of the index_buffers is at args[5]
```

Comment:
Not an issue in this PR, but I do think this stuff is kinda sketchy lol

Reply:
As the author of it I hate it and want to burn it with fire lol